subworkflows/local/arriba_workflow/main.nf

```diff
@@ -0,0 +1,94 @@
+include { ARRIBA_ARRIBA                               } from '../../../modules/nf-core/arriba/arriba/main'
+include { SAMTOOLS_INDEX as SAMTOOLS_INDEX_FOR_ARRIBA } from '../../../modules/nf-core/samtools/index/main'
+include { SAMTOOLS_SORT as SAMTOOLS_SORT_FOR_ARRIBA   } from '../../../modules/nf-core/samtools/sort/main'
+include { SAMTOOLS_VIEW as SAMTOOLS_VIEW_FOR_ARRIBA   } from '../../../modules/nf-core/samtools/view/main'
+include { STAR_ALIGN as STAR_FOR_ARRIBA               } from '../../../modules/nf-core/star/align/main'
+
+workflow ARRIBA_WORKFLOW {
+    take:
+        reads                           // channel [ meta, [ fastqs ]         ]
+        ch_gtf                          // channel [ meta, path_gtf           ]
+        ch_fasta                        // channel [ meta, path_fasta         ]
+        ch_starindex_ref                // channel [ meta, path_index         ]
+        ch_arriba_ref_blacklist         // channel [ meta, path_blacklist     ]
+        ch_arriba_ref_known_fusions     // channel [ meta, path_known_fusions ]
+        ch_arriba_ref_cytobands         // channel [ meta, path_cytobands     ]
+        ch_arriba_ref_protein_domains   // channel [ meta, path_proteins      ]
+        arriba                          // boolean
+        all                             // boolean
+        fusioninspector_only            // boolean
+        star_ignore_sjdbgtf             // boolean
+        seq_center                      // string
+        arriba_fusions                  // path?
+        cram                            // array?
+
+    main:
+        ch_versions = Channel.empty()
+        ch_dummy_file = file("$baseDir/assets/dummy_file_arriba.txt", checkIfExists: true)
+
+        if (( arriba || all) && !fusioninspector_only) {
+
+            STAR_FOR_ARRIBA(
+                reads,
+                ch_starindex_ref,
+                ch_gtf,
+                star_ignore_sjdbgtf,
+                '',                       // seq_platform, should be a params like other pipelines
```

Comment:
should we add `params.seq_platform` like other pipelines?

Reply:
It seems that it would make sense, but probably best done on a separate ticket. Please add the string "TODO" to the comment so that all of these changes are easier to find and we avoid missing any of them.